packages/file/src/types/krb5_conf.coffee.md

```diff
@@ -0,0 +1,88 @@
+
+`nikita.file.types.krb5_conf`
+
+Modify the client Kerberos configuration file located by default in
+"/etc/krb5.conf". Kerberos is a network authentication protocol. It is designed
+to provide strong authentication for client/server applications by using
+secret-key cryptography.
+
+## Schema
+
+    schema =
+      type: 'object'
+      properties:
+        'rootdir':
+          type: 'string'
+          description: """
+          Path to the mount point corresponding to the root directory, optional.
+          """
+        'backup':
+          type: ['string','boolean']
+          description: """
+          Create a backup, append a provided string to the filename extension or
+          a timestamp if value is not a string, only apply if the target file
+          exists and is modified.
+          """
+        'clean':
+          type: 'boolean'
+          description: """
+          Remove all the lines whithout a key and a value, default to "true".
+          """
+        'content':
+          type: 'object'
+          description: """
+          Object to stringify.
+          """
+        'merge':
+          type: 'boolean'
+          description: """
+          Read the target if it exists and merge its content.
+          """
+        'target':
+          type: 'string'
+          description: """
+          Destination file.
+          """
+      required: ['content']
+
+## Example registering a new realm
```

Comment:
Please follow the guideline shared everywhere across the source code, schema comes after examples.